src/sass/facility-locator.scss

```diff
@@ -0,0 +1,10 @@
+@import 'style';
+
+h1{
+  color: grey;
```

Comment:
I realize this is still a first pass but just wanted to point out that there is a core VA stylesheet that has variables for colors, sizes, etc. that should be used when possible.